www/_template/guides/svelte.md

```diff
@@ -1,12 +1,292 @@
+          Learn Svelte
+```
+
+Image: Site showing the new logo
+
+With Svelte, CSS can go directly in your `.svelte` component. Add this code to the top of `App.svelte` between the `<style>` tags:
+
+```css
+<style>
+  .App-header {
+    background-color: #f9f6f6;
+    color: #333;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+  }
+  .App-logo {
+    height: 36vmin;
+    pointer-events: none;
+    margin-bottom: 3rem;
+    animation: App-logo-spin infinite 1.6s ease-in-out alternate;
+  }
+  @keyframes App-logo-spin {
+    from {
+      transform: scale(1);
+    }
+    to {
+      transform: scale(1.06);
+    }
+  }
+</style>
+```
+
+> 💡 Tip: Snowpack has built-in support for [CSS Modules](https://www.snowpack.dev/#import-css-modules) and if you'd like to use Sass there is an official [Sass Plugin](https://www.snowpack.dev/#sass).
+
+Image: Gif showing the results with the logo animated
+
+## Expanding your Svelte component
+
+A previous step demonstrated the built in Hot Module Replacement (HMR) from `@snowpack/plugin-svelte`, but what about Fast Refresh? Snowpack is one of the only Svelte dev environments to support Fast Refresh by default. As you make changes to `.svelte` files, Snowpack will automatically update the browser for you without losing your place or resetting component state. That's Fast Refresh in action. To see this for yourself, go ahead and add a simple timer to your App.svelte component:
+
+Svelte components include component specific scripts in a `<script>` tag. Add the counter here in `App.svelte` between the `<scripe>` tags:
 
 ```js
-// snowpack.config.json
-"plugins": ["@snowpack/plugin-svelte"]
+<script>
+  import {onMount} from 'svelte';
+  let count = 0;
+  onMount(() => {
+    const interval = setInterval(() => count++, 1000);
+    return () => {
+      clearInterval(interval);
+    };
+  });
+</script>
+```
+
+Then lower down in your component's body, add this code that displays the results of the timer.
+
+```diff
+<div class="App">
+    <header class="App-header">
+      <img src="/logo.svg" class="App-logo" alt="logo" />
++     <p>Page has been open for <code>{count}</code> seconds.</p>
+      <a class="App-link" href="https://svelte.dev" target="_blank" rel="noopener noreferrer">
+        Learn Svelte
+      </a>
+    </header>
+</div>
+```
+
+Trying making a change to the code. You'll see the timer does not reset.
+
+Image: GIF showing code and site side by side, when the word "Hello" is added to the .svelte page and the code is saved, the change shows up in the browser without the timer resetting (it keeps counting)
+
+What about other files? Like if you had a file with a regular `.js` extension? For non-`.svelte` files you'll need to add this code snippet to the bottom of `index.js` to get full Hot Module Replacement:
```

Comment:
> What about other, non-Svelte files like `src/index.js`? To re-render your Svelte application when other files change, add this code snippet to he bottom...

^ I like this a bit better because it explains a bit more about what the code is trying to do